src/lift.js

```diff
@@ -4,7 +4,7 @@ var liftN = require('./liftN');
 
 /**
  * "lifts" a function of arity > 1 so that it may "map over" an Array or other
- * Functor.
+ * object that satisfies the FantasyLand Apply spec.
```

Comment:
per the comments, please add a link to "FantasyLand Apply spec": https://github.com/fantasyland/fantasy-land#apply